docs/docs/APIGuide/Layers/Recurrent-Layers.md

```diff
@@ -164,12 +164,12 @@ RecurrentDecoder module is a container of rnn cells which used to make
 a prediction of the next timestep based on the prediction we made from
 the previous timestep.
 
-Input for RecurrentDecoder has to be batch x ???(depends on cell type) without time information. 
+Input for RecurrentDecoder has to be batch x recoderShape without time information. 
```

Comment:
batch x recordShape, where stepShape if the shape of the input of a single time step.